lib/jormungandr/src/Cardano/Wallet/Jormungandr/Api.hs

```diff
@@ -92,7 +94,7 @@ type GetTipId
     :> "tip"
     :> Get '[Hex] BlockId
 
-type PostSignedTx
+type PostMessage
     = "v0"
     :> "message"
     :> ReqBody '[JormungandrBinary] (Tx, [TxWitness])
```

Comment:
Feels a bit weird though that we do `postMesage`, but the body has to be a signed transaction... Okay for now though...